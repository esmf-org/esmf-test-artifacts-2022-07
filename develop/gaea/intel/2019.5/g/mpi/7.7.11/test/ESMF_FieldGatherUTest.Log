build time -- 2022-06-21 00:49:54
20220621 011033.621 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220621 011033.625 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220621 011033.626 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220621 011033.626 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220621 011033.626 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220621 011033.626 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220621 011033.626 INFO             PET0 Running with ESMF Version   : v8.4.0b01-2-gf0d4ab211d
20220621 011033.626 INFO             PET0 ESMF library build date/time: "Jun 21 2022" "00:47:45"
20220621 011033.626 INFO             PET0 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/intel_2019.5_mpi_g_develop
20220621 011033.627 INFO             PET0 ESMF_COMM                   : mpi
20220621 011033.780 INFO             PET0 ESMF_MOAB                   : enabled
20220621 011033.780 INFO             PET0 ESMF_LAPACK                 : enabled
20220621 011033.780 INFO             PET0 ESMF_NETCDF                 : enabled
20220621 011033.780 INFO             PET0 ESMF_PNETCDF                : disabled
20220621 011033.780 INFO             PET0 ESMF_PIO                    : enabled
20220621 011033.780 INFO             PET0 ESMF_YAMLCPP                : enabled
20220621 011033.919 INFO             PET0  Beginning Test, file ESMF_FieldGatherUTest.F90, line 58
20220621 011033.920 INFO             PET0  NUMBER_OF_PROCESSORS           4
20220621 011033.983 INFO             PET0  PASS  FieldGather 1d test, ESMF_FieldGatherUTest.F90, line 72
20220621 011033.995 INFO             PET0  PASS  FieldGather 2d test, ESMF_FieldGatherUTest.F90, line 80
20220621 011034.007 INFO             PET0  PASS  FieldGather 3d test, ESMF_FieldGatherUTest.F90, line 88
20220621 011034.013 INFO             PET0  PASS  FieldScatter 2d test, ESMF_FieldGatherUTest.F90, line 96
20220621 011034.013 INFO             PET0  Ending Test, file ESMF_FieldGatherUTest.F90, line 99
20220621 011034.013 INFO             PET0 Finalizing ESMF
20220621 011033.624 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220621 011033.625 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220621 011033.626 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220621 011033.626 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220621 011033.626 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220621 011033.626 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220621 011033.626 INFO             PET1 Running with ESMF Version   : v8.4.0b01-2-gf0d4ab211d
20220621 011033.626 INFO             PET1 ESMF library build date/time: "Jun 21 2022" "00:47:45"
20220621 011033.626 INFO             PET1 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/intel_2019.5_mpi_g_develop
20220621 011033.627 INFO             PET1 ESMF_COMM                   : mpi
20220621 011033.886 INFO             PET1 ESMF_MOAB                   : enabled
20220621 011033.886 INFO             PET1 ESMF_LAPACK                 : enabled
20220621 011033.886 INFO             PET1 ESMF_NETCDF                 : enabled
20220621 011033.886 INFO             PET1 ESMF_PNETCDF                : disabled
20220621 011033.886 INFO             PET1 ESMF_PIO                    : enabled
20220621 011033.886 INFO             PET1 ESMF_YAMLCPP                : enabled
20220621 011033.919 INFO             PET1  Beginning Test, file ESMF_FieldGatherUTest.F90, line 58
20220621 011033.920 INFO             PET1  NUMBER_OF_PROCESSORS           4
20220621 011033.983 INFO             PET1  PASS  FieldGather 1d test, ESMF_FieldGatherUTest.F90, line 72
20220621 011033.995 INFO             PET1  PASS  FieldGather 2d test, ESMF_FieldGatherUTest.F90, line 80
20220621 011034.007 INFO             PET1  PASS  FieldGather 3d test, ESMF_FieldGatherUTest.F90, line 88
20220621 011034.013 INFO             PET1  PASS  FieldScatter 2d test, ESMF_FieldGatherUTest.F90, line 96
20220621 011034.013 INFO             PET1  Ending Test, file ESMF_FieldGatherUTest.F90, line 99
20220621 011034.013 INFO             PET1 Finalizing ESMF
20220621 011033.624 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220621 011033.625 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220621 011033.626 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220621 011033.626 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220621 011033.626 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220621 011033.626 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220621 011033.626 INFO             PET2 Running with ESMF Version   : v8.4.0b01-2-gf0d4ab211d
20220621 011033.626 INFO             PET2 ESMF library build date/time: "Jun 21 2022" "00:47:45"
20220621 011033.626 INFO             PET2 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/intel_2019.5_mpi_g_develop
20220621 011033.627 INFO             PET2 ESMF_COMM                   : mpi
20220621 011033.917 INFO             PET2 ESMF_MOAB                   : enabled
20220621 011033.918 INFO             PET2 ESMF_LAPACK                 : enabled
20220621 011033.918 INFO             PET2 ESMF_NETCDF                 : enabled
20220621 011033.918 INFO             PET2 ESMF_PNETCDF                : disabled
20220621 011033.918 INFO             PET2 ESMF_PIO                    : enabled
20220621 011033.918 INFO             PET2 ESMF_YAMLCPP                : enabled
20220621 011033.919 INFO             PET2  Beginning Test, file ESMF_FieldGatherUTest.F90, line 58
20220621 011033.920 INFO             PET2  NUMBER_OF_PROCESSORS           4
20220621 011033.983 INFO             PET2  PASS  FieldGather 1d test, ESMF_FieldGatherUTest.F90, line 72
20220621 011033.995 INFO             PET2  PASS  FieldGather 2d test, ESMF_FieldGatherUTest.F90, line 80
20220621 011034.007 INFO             PET2  PASS  FieldGather 3d test, ESMF_FieldGatherUTest.F90, line 88
20220621 011034.013 INFO             PET2  PASS  FieldScatter 2d test, ESMF_FieldGatherUTest.F90, line 96
20220621 011034.013 INFO             PET2  Ending Test, file ESMF_FieldGatherUTest.F90, line 99
20220621 011034.013 INFO             PET2 Finalizing ESMF
20220621 011033.621 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220621 011033.625 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220621 011033.626 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220621 011033.626 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220621 011033.626 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220621 011033.626 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220621 011033.626 INFO             PET3 Running with ESMF Version   : v8.4.0b01-2-gf0d4ab211d
20220621 011033.626 INFO             PET3 ESMF library build date/time: "Jun 21 2022" "00:47:45"
20220621 011033.626 INFO             PET3 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/intel_2019.5_mpi_g_develop
20220621 011033.627 INFO             PET3 ESMF_COMM                   : mpi
20220621 011033.629 INFO             PET3 ESMF_MOAB                   : enabled
20220621 011033.629 INFO             PET3 ESMF_LAPACK                 : enabled
20220621 011033.629 INFO             PET3 ESMF_NETCDF                 : enabled
20220621 011033.629 INFO             PET3 ESMF_PNETCDF                : disabled
20220621 011033.629 INFO             PET3 ESMF_PIO                    : enabled
20220621 011033.629 INFO             PET3 ESMF_YAMLCPP                : enabled
20220621 011033.919 INFO             PET3  Beginning Test, file ESMF_FieldGatherUTest.F90, line 58
20220621 011033.920 INFO             PET3  NUMBER_OF_PROCESSORS           4
20220621 011033.983 INFO             PET3  PASS  FieldGather 1d test, ESMF_FieldGatherUTest.F90, line 72
20220621 011033.995 INFO             PET3  PASS  FieldGather 2d test, ESMF_FieldGatherUTest.F90, line 80
20220621 011034.007 INFO             PET3  PASS  FieldGather 3d test, ESMF_FieldGatherUTest.F90, line 88
20220621 011034.013 INFO             PET3  PASS  FieldScatter 2d test, ESMF_FieldGatherUTest.F90, line 96
20220621 011034.013 INFO             PET3  Ending Test, file ESMF_FieldGatherUTest.F90, line 99
20220621 011034.013 INFO             PET3 Finalizing ESMF
