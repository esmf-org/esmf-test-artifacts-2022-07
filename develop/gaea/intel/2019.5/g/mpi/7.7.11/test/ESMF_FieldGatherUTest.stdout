build time -- 2022-07-15 00:48:09
srun: error: get_addr_info: getaddrinfo() failed: Name or service not known
srun: error: slurm_set_addr: Unable to resolve "c4-slurm-backup"
srun: error: slurm_set_port: attempting to set port without address family
srun: error: get_addr_info: getaddrinfo() failed: Name or service not known
srun: error: slurm_set_addr: Unable to resolve "c4-slurm-backup"
srun: error: slurm_set_port: attempting to set port without address family
  Beginning Test, file ESMF_FieldGatherUTest.F90, line 58
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_FieldGatherUTest.F90, line 58
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_FieldGatherUTest.F90, line 58
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_FieldGatherUTest.F90, line 58
  NUMBER_OF_PROCESSORS           4
  PASS  FieldGather 1d test, ESMF_FieldGatherUTest.F90, line 72
  PASS  FieldGather 1d test, ESMF_FieldGatherUTest.F90, line 72
  PASS  FieldGather 1d test, ESMF_FieldGatherUTest.F90, line 72
  PASS  FieldGather 1d test, ESMF_FieldGatherUTest.F90, line 72
  PASS  FieldGather 2d test, ESMF_FieldGatherUTest.F90, line 80
  PASS  FieldGather 2d test, ESMF_FieldGatherUTest.F90, line 80
  PASS  FieldGather 2d test, ESMF_FieldGatherUTest.F90, line 80
  PASS  FieldGather 2d test, ESMF_FieldGatherUTest.F90, line 80
  PASS  FieldGather 3d test, ESMF_FieldGatherUTest.F90, line 88
  PASS  FieldGather 3d test, ESMF_FieldGatherUTest.F90, line 88
  PASS  FieldGather 3d test, ESMF_FieldGatherUTest.F90, line 88
  PASS  FieldGather 3d test, ESMF_FieldGatherUTest.F90, line 88
  PASS  FieldScatter 2d test, ESMF_FieldGatherUTest.F90, line 96
  PASS  FieldScatter 2d test, ESMF_FieldGatherUTest.F90, line 96
  Ending Test, file ESMF_FieldGatherUTest.F90, line 99
  PASS  FieldScatter 2d test, ESMF_FieldGatherUTest.F90, line 96
  Ending Test, file ESMF_FieldGatherUTest.F90, line 99
  PASS  FieldScatter 2d test, ESMF_FieldGatherUTest.F90, line 96
  Ending Test, file ESMF_FieldGatherUTest.F90, line 99
  Ending Test, file ESMF_FieldGatherUTest.F90, line 99
  PET           2  Test Elapsed Time    43.3490000000000       msec.
  PET           1  Test Elapsed Time    43.6790000000000       msec.
  PET           0  Test Elapsed Time    8.86700000000000       msec.
  PET           3  Test Elapsed Time    43.6450000000000       msec.
